pkgs/development/python-modules/openapi-python-client-integration-tests/default.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  attrs,
+  httpx,
+  poetry-core,
+  python-dateutil,
+}:
+
+buildPythonPackage rec {
+  pname = "openapi-python-client-integration-tests";
+  version = "0.14.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "openapi-generators";
+    repo = "openapi-python-client";
+    rev = "v${version}";
+    hash = "sha256-Etw0Iz5aAdu/K+WSmue9ux8qi0f/YKgZgVhqCbGVxE0=";
+  };
+
+  sourceRoot = "source/integration-tests";
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace "poetry.masonry.api" "poetry.core.masonry.api"
+  '';
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    attrs
+    httpx
+    python-dateutil
+  ];
+
```

```suggestion

  pythonImportsCheck = [ "openapi_python_client" ];

```

Missing pythonImportsCheck. Can you double check that the other packages have them, too?